instrumentation/jsonrpc4j-1.3/library/src/main/java/io/opentelemetry/instrumentation/jsonrpc4j/v1_3/JsonRpcClientAttributesGetter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jsonrpc4j.v1_3;
+
+import io.opentelemetry.instrumentation.api.incubator.semconv.rpc.RpcAttributesGetter;
+
+// Check
+// https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-metrics.md#attributes
+// Check https://opentelemetry.io/docs/specs/semconv/rpc/json-rpc/
+public enum JsonRpcClientAttributesGetter implements RpcAttributesGetter<JsonRpcClientRequest> {
+  INSTANCE;
+
+  @Override
+  public String getSystem(JsonRpcClientRequest request) {
+    return "jsonrpc";
+  }
+
+  @Override
+  public String getService(JsonRpcClientRequest request) {
+    if (request.getMethod() != null) {
+      return request.getMethod().getDeclaringClass().getName();
+    }
+    return "NOT_AVAILABLE";
```

Comment:
is there anything in the spec that says you should use `NOT_AVAILABLE ` when service can not be determined?

Reply:
Or should I leave it as `null` ?

Reply:
I tend to set it as `null` , as the `rpc.service` is not a required attribute according to the [spec](https://opentelemetry.io/docs/specs/semconv/rpc/rpc-spans/):
<img width="715" alt="image" src="https://github.com/user-attachments/assets/8a66dd1a-ad56-42ef-a9fe-a4691f73e610" />